ano/environment.py

```diff
@@ -251,7 +254,7 @@ def board_models(self):
         # - hardware/arduino/{chipset}/boards.txt (Arduino 1.5.x, chipset like `avr`, `sam`)
         # - hardware/{platform}/boards.txt (MPIDE 0.xx, platform like `arduino`, `pic32`)
         # we should find and merge them all
-        boards_txts = self.find_arduino_file('boards.txt', ['hardware', '**'], 
+        boards_txts = self.find_arduino_file('boards.txt', ['**', 'hardware', '**'],
```

Comment:
I'm missing something with this change: '**' is already in the array. How is this different?